cli/flox/doc/doc-compose/manifest.toml.md

```diff
@@ -0,0 +1,583 @@
+    environment.
+
+`shutdown.command`
+:   A command to run to shut down the service instead of delivering the SIGTERM
+    signal to the process. Some programs require special handling to shut down
+    properly e.g. a program that spawns a server process and uses a client to
+    tell the server to shut down. Sending a SIGTERM to a client in that case
+    may not shut down the server. In those cases you may provide a specific
+    shutdown command to run instead of relying on the default behavior of
+    sending a SIGTERM to the service. This field is required if the `is-daemon`
+    field is `true`.
+
+`systems`
+:   An optional list of systems on which to run this service.
+    If omitted, the service is not restricted.
+
+## `[include]`
+
+The `[include]` section of the manifest describes other environments that you'd
+like to merge with the current manifest in order to compose them into a single
+environment.
+
+The list of environments to include is specified by the `include.environments`
+array. The order of the "include descriptors" in this array specifies the
+priority that should be used when merging the manifests. Descriptors later in
+the array take higher priority than those earlier in the array, and manifest
+fields in the composing manifest take the highest priority.
+
+When merging manifests, different sections have different merge semantics.
+
+`[install]`
+: Package descriptors are overwritten entirely by a higher priority manifest.
+
+`[vars]`
+: Variables are overwritten entirely by a higher priority manifest.
+
+`[hook]`
+: The scripts in `hook` are appended to one another with a newline in betweeen.
+  Scripts from higher priority manifests come after those from lower priority
+  manifests.
+
+`[profile]`
+: The scripts in the `profile` section are appended in the same way that they
+  are for `hook`.
+
+`[services]`
+: Service descriptors are entirely overwritten by higher priority manifests
+
+`[include]`
+: The `include` section is ommitted from merged manifests, so no merging of the
+  `include` section ever happens.
+
+`[containerize]`
+: The `containerize.config` field is deep merged, meaning that individual
+  fields of `containerize.config` are merged rather than `containerize.config`
+  being completely overwritten. The fields within `containerize.config` are
+  merged as follows: `user`, `cmd`, `working_dir`, and `stop_signal` are
+  overwritten; `labels` and `exposed_ports` are merged via the union of the
+  values in the high priority and low priority manifests.
+
+`[options]`
+: The `options` section is also deep merged, meaning that individual fields of
+  the `options` section are merged rather than being completely overwritten.
+  All of the fields in the `options` section are individually overwritten by
+  higher priority manifests e.g. `options.allow.broken` is individually
+  overwritten by a higher priority manifest, as is `options.allow.licenses`,
+  etc.
```

Comment:
(reviewer commentary) This is the new section in case anyone else was looking at the wall of green trying to figure out what changed 😄